### DISCUSSION TITLE
TreeView [v. 3.5.14] load children async

### DISCUSSION THREAD

**Discussion by BettaKalpa** [vuetifyjs/vuetify#19566] [Q&A]
What is wrong on this code?

https://play.vuetifyjs.com/#eNqtVU1v4zYQ/SuEUMAOYEk2sjnUsIPNtjlsD92g3fYSB1haGtvcUiJBUnYMQ/+9w6G+Ymd3c1ggiKWZN1+PT8PHU2RNlt5pnewriObRwkGhJXdwuyoZW+zjjJucnru32AkngWWSW7tcRettLMpcbBVz8Oziw044CI+7m1XUhDL2LzeiUpVlvwsDmVPm2CRNh1nbSrnY337SUEI+Z6cTU/jI6nqRevsAcpc5scdmA4r7NxjguradAdgLOLTN7ONC5SDnFOHjcY4QvYrOMb42AfxD757jmIWfn34Hdql4Hmc7IXMDJfo34LLdfQ/IlFQG7QduSlFue0cOpRXuiK5MFRrb6V2+RiAIvfR75ttzWXmfyHtHMxxfI3pogtg6g0NvfSmLLUjwJwg4HrYwqIoo35FQDZHdWXYiQZKsVG7OtQ/GdCdqh9WDg6cDEJkqb1mRixgpBQz0x07Gb8AwXybkJWyRvhBo0E97vEGzjaDw5QUYX21mhHbMgqs0YUWhlXHshIdS6Ap1MGEGNqxmG6MKNsJPYuRh2IB1TPPKAluywrLlLSvhwB4QJSyMDVglUXloxtyfRQGqcq11ggFXV0GMIVGj06WvNX58Ql/rIZ2/YieNDR29awvuY+Pl9lhmbHzl+zgFdviBi6bz8exmOqWcQTM2IdFg3GNLbhNECPykZtPpdNKbSHdzNnpQ1tnRwNGKfc4enzpz3T1dZp29kvVOrqvi7Wmf/E89JCJ8Zx0NpMMfEuFDlYREqu3YRzTUYqcbRoYkLLvlctkOftUPFJJS4Q/Hf4ykiAkb7ZzTdp6mX60qUX4Z7JTMwSTuqFHMOSQot1SHbCFXzUCiuH5OZh6obFO3I53P2giAYJdMUtleVKF+ZeRQXc4c+5ZDLIreB/Xtj7/8csKw+kvTTgv0E3RIjEq8YdyBiPtWAYmu7G6cJITpGMs4pmdjMGZwJO2QfruSqyOhG7L9YnyxRRp2Am6DaBI5i+EbsaU6eBlS1rCOhQTzSftdiLseb5uQFm8NKdXhD7I5U0GjTozZQfbfK/av9tnbVtEDzgzGXzidz3GDvQX3/d9/4hU6cOJlVPm1/x3nX37hVLSvCfahKnNse4Cjbj/S0sO9/9nePzt/6TRD+UaJJ8KvIlx/v31n9L7d6+S64Rf/kMewVuOC6zMmg+NlGqwS2m21neUlhuE9IfYmKcGlpS7S9whLTVU6XK5xror318m7ZPYrXvTWDe0J2CJeG3VAcjHLYHZM4MTm+IZaDRJL3CSzd6FEY4slX1tf4yJ3SqdpYpQrfpCe9bfNdBb2Yq4z38VsA9LrSVlJSf+e/geHJi3u

As you can see you can load only one parent's children at time

**Comment by johnleider** [comment]
Fixed in https://github.com/vuetifyjs/vuetify/pull/19773
